### DISCUSSION TITLE
Bildpfade im Datalinq

### DISCUSSION THREAD

**Discussion by ue-black** [e-netze/datalinq-community#6] [General]
Hallo!

Ich möchte in meiner Datalinq Abfrage gerne Bilddateien einfügen. Ich habe in der Doku nirgends ein Beispiel für einen relativen Bildpfad gefunden bzw. wo ich die jpg/pngs ablegen kann.

Vielen Dank für die Hilfe!
LG Klaus

**Comment by gis-eni** [comment]
Hallo Klaus!

Datalinq bietet eigentlich keinen offiziellen Pfad für Bilder, die du in einem DataLinq View anzeigen kannst. Dafür ist es eigentlich nicht gemacht. Für solche Dinge machen wir uns in der Regel ein Virtuelles Verzeichnis im IIS und legen dort unsre statischen Files ab.

Du kannst die Dateien auch bei der WebGIS API oder im WebGIS Portal irgendwo unter `wwwroot` ablegen. Statisch Files von können in der Regel auch vom Browser abgeholt werden. Allerdings musst du dann immer bei einem Update aufpassen, dass die Files mit kopiert werden.

LG Jürgen
